### PR TITLE
build: update tdvf to 2022-tdvf-ww39.4

### PR DESCRIPTION
**Pull request by jialeif**
Use default submodule url for cmocka.

Signed-off-by: Feng, Jialei <jialei.feng@intel.com>

### CODE REVIEW

**Review by kenplusplus — APPROVED** [review]
LGTM

File: build/ubuntu-22.04/intel-mvp-tdx-tdvf/debian/changelog [hunk 1]
```diff
@@ -1,3 +1,31 @@
...
+
+  * TDX build.
+    Installed to /usr/share/tdvf/
+
+ -- Jialei Feng <jialei.feng@intel.com>  Wed, 17 Aug 2022 11:44:00 +0800
+
+tdx-tdvf (2022.8.8-upstream1.mvp39) jammy; urgency=medium
+
+  * TDX build.
+    Installed to /usr/share/tdvf/
```
Comment (kenplusplus) [inline build/ubuntu-22.04/intel-mvp-tdx-tdvf/debian/changelog:25]: Duplicate with line 15

File: build/ubuntu-22.04/intel-mvp-tdx-tdvf/debian/changelog [hunk 2]
```diff
@@ -1,3 +1,31 @@
...
+
+  * TDX build.
+    Use OvmfPkg/IntelTdx/IntelTdxX64.dsc
+
+ -- Jialei Feng <jialei.feng@intel.com>  Mon, 26 Sep 2022 13:06:00 +0800
+
+tdx-tdvf (2022.8.17-upstream2.mvp41) jammy; urgency=medium
+
+  * TDX build.
+    Use OvmfPkg/IntelTdx/IntelTdxX64.dsc
```
Comment (kenplusplus) [inline build/ubuntu-22.04/intel-mvp-tdx-tdvf/debian/changelog:11]: Duplicate with line 4

File: build/ubuntu-22.04/intel-mvp-tdx-tdvf/debian/changelog (hunk 1 above)
Comment (jialeif) [reply build/ubuntu-22.04/intel-mvp-tdx-tdvf/debian/changelog]: Thanks updated

**Pull request merged**